_posts/2017-10-23-survey-focus-group-update.md

```diff
@@ -0,0 +1,75 @@
+
+Lead by [Kari L. Jordan](https://github.com/kariljordan), the focus group included [Stephen Childs](https://github.com/sechilds), [Alycia Crall](https://twitter.com/alycrall), [Reshama Shaikh](https://twitter.com/reshamas), [Aleksandra Nenadic](https://github.com/anenadic), [Karen Word](http://twitter.com/karen_word), and [Louisa Bells](https://twitter.com/ChemCurator). These individuals represent have expertise in data science assessment and evaluation. 
+
+The focus group’s first [meeting](https://docs.google.com/document/d/16GEJiHNSla0aE1oCRjqDRyLlIz1q2NCXFJt3vWs70zw/edit) was Friday, October 20th 2017. The primary goal of this meeting was to determine how we could assign unique identifiers to our learners and still have their data remain confidential. Additionally, we discussed the need to collect demographic information, and how to accomplish that in an inclusive way.
+
+In terms of unique identifiers, an idea was to ask learners to provide their personal e-mail address as their unique identifier. Issues were raised around learners not remembering which e-mail address they gave, and that by providing their e-mail address their responses would not be anonymous. We discussed stating in the introductory language that responses would be *confidential*, but not *anonymous* as research staff (Kari Jordan and Erin Becker) have access to the data. There was also the idea of having SurveyMonkey create unique survey links for participants, but this would not be feasible because our survey links are embedded into the [workshop website template](https://github.com/datacarpentry/workshop-template). SurveyMonkey does offer the ability to create [custom variables](https://help.surveymonkey.com/articles/en_US/kb/What-are-custom-variables-and-how-do-I-use-them) however, this will add more manual work for staff and workshop hosts. A graphic was also discussed as an option for unique identifiers, however, we are concerned that learners may not remember what graphic they chose. We are still searching for a solution, so if you have ideas please [get in touch](mailto:kariljordan@carpentries.org). 
+
+U.S. based workshop participants are asked demographic questions (race/ethnicity, gender, etc.) Collecting this type of information can be challenging, and we want to be sure we are asking demographic questions in an inclusive manner. Thus, instead of asking *what is your gender*, and having learners select from the following, 
+
+          + Female 
+          + Male 
+          + Other 
+          + Prefer not to say 
+
+the *gender* question was changed to a free response question where learners are able to type in how they identify themselves. This will result in more data cleaning, but this can easily be done in OpenRefine.
```

Comment:
In the over 10,000 responses to the SWC pre-workshop survey, only 29 respondents (0.35%) chose "Other". I *strongly* advise against making this question a pure open-response as we're going to get a lot of variability in how people decide to enter the two most popular answers (male and female). For example, for "female" we're going to get "F", "Female", "female", "woman", "f", and probably some variations and typos on those responses.  It is very important that we be able to report our gender distribution for funding and other metrics, so making this more difficult is not recommended.

Reply:
Agree we should update this question. However, we shouldn't have it as free-response, as Erin mentioned for doing analysis of the data. There are best practices in asking gender identity questions. Let's find those best practices and use them here.

Reply:
I'll update the question as:
To which gender do you most identify:
+ Female
+ Male
+ Transgender female
+ Transgender male
+ Gender variant/non-conforming
+ Not listed
+ Prefer not to answer